### PR TITLE
Deploy everypolitician.org from Travis

### PR DESCRIPTION
**Pull request by chrismytton**
This adds a script to deploy http://everypolitician.org that runs each time Travis successfully builds the `master` branch.

It does this by installing [viewer-sinatra](https://github.com/everypolitician/viewer-sinatra) and using it to rebuild [viewer-static](https://github.com/everypolitician/viewer-static).

Related to https://github.com/everypolitician/everypolitician/issues/506 as part of [changing how everypolitician.org is deployed](https://github.com/everypolitician/everypolitician/projects/2).

### CODE REVIEW

**Comment by chrismytton** [comment]
@tmtmtmtm 👀

File: scripts/deploy.sh [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+}
+
+build_viewer_static() {
+  cd /tmp
+  wget -nv -m localhost:4567/status/all_countries.html || (echo "wget exited with non-zero exit code: $?" >&2 && exit 1)
+}
+
+deploy_viewer_static() {
+  cd /tmp
+  git clone https://github.com/everypolitician/viewer-static.git
```
Comment (tmtmtmtm) [inline scripts/deploy.sh]: Would we be better with a shallow clone here?

Comment (chrismytton) [inline scripts/deploy.sh]: Ah yes, that's a very good idea. Amazingly, according to git, a shallow clone with `--depth 1` _still_ results in a 13.68 MiB clone, but I suppose that's much better than 700+MB! Will add that now.

**Comment by chrismytton** [comment]
@tmtmtmtm I've addressed your comments so I think this is ready for another review now.

**Comment by tmtmtmtm** [comment]
Excellent. We presumably also want to point back to this one from there as well — I'm assuming we'll also want to simplify that one to use the same deploy mechanism too, rather checking TRAVIS_PULL_REQUEST etc? (assuming so, we should probably have an Issue for that, if there isn't one already)

**Pull request merged**